tests/regression_tests/dagmc/tallies.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<tallies>
+
+  <filter id="1" type="cell">
+    <bins>1</bins>
+  </filter>
+
+  <filter id="2" type="energy">
+    <bins>0 20.0e6</bins>
+  </filter>
+
+  <filter id="3" type="energyout">
+    <bins>0 20.0e6</bins>
+  </filter>
+
+  <tally id="1">
+    <filters>1</filters>
+    <scores>total scatter nu-scatter absorption fission nu-fission</scores>
+  </tally>
+
+  <tally id="2">
+    <filters>1 2</filters>
+    <scores>total scatter nu-scatter absorption fission nu-fission</scores>
+  </tally>
+
+  <tally id="3">
+    <filters>1 2 3</filters>
+    <scores>scatter nu-scatter nu-fission</scores>
+  </tally>
```

Comment:
No need to test all of these tallies.  Probably just a single tally with filter 1 and a `total` score will do.